cfme/tests/networks/nuage/test_automation.py

```diff
@@ -0,0 +1,11 @@
+def test_authentication(appliance):
```

Comment:
This needs a docblock describing the test, and likely a more descriptive method name. Its not clear to me how this is testing authentication? Looks like a test of the default automation schema fields?

Does a nuage provider need to be configured in order to run this test?  Is it relevant against MIQ and downstream CFME versions?